.travis.yml

```diff
@@ -18,6 +18,15 @@ jobs:
       script: sbt ci-test
     - stage: release
       script: sbt ci-release
+    - env:
```

Comment:
not sure about introducing a second job.
Maybe it's better if we publish the website only if the release succeeds?

Reply:
Let's start with this, maybe we'll join the two later.